pkg/skaffold/build/gcb/buildpacks.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gcb
+
+import (
+	cloudbuild "google.golang.org/api/cloudbuild/v1"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+)
+
+func (b *Builder) buildpackBuildSpec(artifact *latest.BuildpackArtifact, tag string) (cloudbuild.Build, error) {
+	steps := []*cloudbuild.BuildStep{
+		{
+			Name: "busybox",
+			Args: []string{"sh", "-c", "chown -R 1000:1000 /workspace /layers $$HOME"},
```

Comment:
Should this be 
```suggestion
			Args: []string{"sh", "-c", "chown -R $$CNB_USER_ID:$$CNB_GROUP_ID /workspace /layers $$HOME"},
```

Reply:
Well, `CNB_USER_ID` is hard coded for now and not available as an env variable on GCB.

Reply:
Oh! Maybe this should use the build image rather than busybox.

Reply:
(Thinking of distroless images where we should use the non-root user, which [I believe is 65532](https://github.com/GoogleContainerTools/distroless/blob/b54513ef989c81d68cb27d9c7958697e2fedd2c4/base/base.bzl#L8).)

Reply:
Excellent idea. I changed that [here](https://github.com/GoogleContainerTools/skaffold/pull/3257)